src/store.ts

```diff
@@ -121,10 +121,16 @@ function createSDKConfig(config: SDKInitConfig): SDKConfig {
 //       to TypeScript
 export type ServerSettings = Dictionary;
 export type SessionAttributes = Dictionary;
-export type IntegrationAttributes = Dictionary<Dictionary<string>>;
 
-type WrapperSDKTypes = 'flutter' | 'none';
-interface WrapperSDKInfo {
+// FIXME: DefinitelyTyped Batch definition defines this as a string but
```

Comment:
I don't think this comment still applies based on my memory of us discussing this, right?.   we pass in a number but it gets set as a string bc JS is JS. the `number` portion is when we are using the method mParticle.setIntegrationAttribute(_moduleid_, 'foo')